mathics/core/characters.py

```diff
@@ -339,7 +339,7 @@
     'Equal': '\uF431',
     'EqualTilde': '\u2242',
     'Equilibrium': '\u21CC',
-    'Equivalent': '\u29E6',
+    'Equivalent': '\u21D4',
```

Comment:
It's true that in mathematica "Equivalent" is "\u29E6" and "Implies" is "\uF523", but in Mathics they look like wrong characters. I searched in the internet and found "\u29E6" was "⧦" and "\uF523" was in the private use area. I think Mathematica uses its own font to show math symbols. That's an awkward problem... I will change characters.py back.